Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
+        /// The prepaid card dispute deposit
         case prepaidCardDisputeDeposit = "PREPAID_CARD_DISPUTE_DEPOSIT"
+        /// The prepaid card domestic cash withdrawal fee
         case prepaidCardDomesticCashWithdrawalFee = "PREPAID_CARD_DOMESTIC_CASH_WITHDRAWAL_FEE"
+        /// The prepaid card exchange rate difference
         case prepaidCardExchangeRateDifference = "PREPAID_CARD_EXCHANGE_RATE_DIFFERENCE"
+        /// The prepaid card manual unload
         case prepaidCardManualUnload = "PREPAID_CARD_MANUAL_UNLOAD"
+        /// The prepaid card overseas cash withdrawal fee
         case prepaidCardOverseasCashWithdrawalFee = "PREPAID_CARD_OVERSEAS_CASH_WITHDRAWAL_FEE"
+        /// The prepaid card pin change fee
         case prepaidCardPinChangeFee = "PREPAID_CARD_PIN_CHANGE_FEE"
+        /// The prepaid card refund
         case prepaidCardRefund = "PREPAID_CARD_REFUND"
+        /// The prepaid card replacement fee
         case prepaidCardReplacementFee = "PREPAID_CARD_REPLACEMENT_FEE"
+        /// The prepaid card sale reversal
         case prepaidCardSaleReversal = "PREPAID_CARD_SALE_REVERSAL"
+        /// The prepaid card unload
         case prepaidCardUnload = "PREPAID_CARD_UNLOAD"
+        /// The transfer to debitcard
         case transferToDebitCard = "TRANSFER_TO_DEBIT_CARD"
+        /// The transfer to prepaid card
         case transferToPrepaidCard = "TRANSFER_TO_PREPAID_CARD"
-        // Donation types
+        /// Donation types
+        ///
+        /// The donation
         case donation = "DONATION"
+        /// The donation fee
         case donationFee = "DONATION_FEE"
+        /// The donation return
         case donationReturn = "DONATION_RETURN"
-        // Merchant payment types
+        /// Merchant payment types
+        ///
+        /// The merchant payment
         case merchantPayment = "MERCHANT_PAYMENT"
+        /// The merchant payment fee
         case merchantPaymentFee = "MERCHANT_PAYMENT_FEE"
+        /// The merchant payment refund
         case merchantPaymentRefund = "MERCHANT_PAYMENT_REFUND"
+        /// The merchant payment return
         case merchantPaymentReturn = "MERCHANT_PAYMENT_RETURN"
-        // MoneyGram types
+        /// MoneyGram types
+        ///
+        /// The money gram transfer return
         case moneygramTransferReturn = "MONEYGRAM_TRANSFER_RETURN"
+        /// The transfer to money gram
         case transferToMoneygram = "TRANSFER_TO_MONEYGRAM"
-        // Paper check types
+        /// Paper check types
+        ///
+        /// The paper check fee
         case paperCheckFee = "PAPER_CHECK_FEE"
+        /// The paper check refund
         case paperCheckRefund = "PAPER_CHECK_REFUND"
+        /// The transfer to paper check
         case transferToPaperCheck = "TRANSFER_TO_PAPER_CHECK"
-        // User and program account types
+        /// User and program account types
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /